dt-groups/groups-post-type.php

```diff
@@ -238,86 +238,119 @@ public function get_group_field_defaults( $post_id = null, $include_current_post
 
         $fields = [];
 
+        $fields["group_name"] = [
```

Comment:
@mikeallbutt please don't add extra fields. This will create confusion in other places.